src/js/components/PopoverComponent.js

```diff
@@ -0,0 +1,81 @@
+var React = require("react/addons");
+var classNames = require("classnames");
+
+var Util = require("../helpers/Util");
+
+const DEFAULT_ALIGNED = "default";
```

Comment:
Can't these be a property like `propTypes.oneOf(["default", "top", "bottom"])` ? I don't see the need for C #defines here :laughing: